src/service/world-api-adapter.js

```diff
@@ -1,11 +1,22 @@
 /**
  * ##World API Adapter
  *
- * The World API Adapter allows you to create, access, and manipulate multiplayer worlds its users and runs
+ * Epicenter includes support for multiplayer games. These are projects where multiple end users share a run and work together in the same "world." Only [team projects](../../../glossary/#team) can be multiplayer.
  *
- * All API calls take in an "options" object as the last parameter. The options can be used to extend/override the World API Service defaults.
+ * The World API Adapter allows you to create, access, and manipulate multiplayer worlds, including end users and runs playing within the world.
  *
+ * All API calls take in an "options" object as the last parameter. The options can be used to extend/override the World API Service defaults.
```

Comment:
Preface with 'As with all the other API Adapters'. Also maybe say 'all methods' instead of 'all api calls'